src/dom/dom.ts

```diff
@@ -980,10 +1025,77 @@ export class dom {
         allTextNodes.forEach((node)=> {
            node.nodeValue = node.nodeValue.replace(/\s+([^\s]*)\s*$/, nbsp + '$1')
         })
+    }
 
+    /**
+     * Add an event listener to multiple events.
+     * Useful for when mouse and touch events should call the same listener.
+     */
+    static addEventListeners(
+        element: Element|Window,
+        events: string[],
+        listener: EventListenerOrEventListenerObject
+    ) {
+        events.forEach(
+            (event) => element.addEventListener(event, listener));
+    }
+
+    /**
+     * Remove an event listener for multiple events.
+     * Useful for when mouse and touch events should call the same listener.
+     */
+    static removeEventListeners(
+        element: Element|Window,
+        events: string[],
+        listener: EventListenerOrEventListenerObject
+    ) {
+        events.forEach(
+            (event) => element.removeEventListener(event, listener));
     }
 
     static getScrollElement(): Element {
         return document.scrollingElement || document.documentElement;
     }
+
+    /**
+     * Determine the distance between the centers of two given elements.
+     * If no second element is given, the root element is used.
```

Comment:
I think we can just say the root is the document to make it clearer.

Reply:
Put `document.children[0]`to be explicit to avoid any confusion.